.github/workflows/test-storybook.yml

```
@@ -0,0 +1,33 @@
+jobs:
+  chromatic-deployment:
+    # Run on main repo pushes or when PR has "deploy-storybook" label
+    if: github.repository == 'rad-ui/ui' && (github.event_name == 'push' || github.event.label.name == 'deploy-storybook')
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0  # Required for Chromatic to retrieve git history
+
+      - name: Use Node.js
+        uses: actions/setup-node@v3
```

actionlint (1.7.4)

the runner of "actions/setup-node@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)